src/check/parse/tokenize.zig

```diff
@@ -1043,23 +1089,27 @@ pub const Tokenizer = struct {
                     if (!self.cursor.chompIdentGeneral()) {
```

Comment:
In the future, I think we'd want this function (and similars like `chompIdentLower`) to calculate attributes as it chomps.

Reply:
Agreed! Wanted to keep it simple for the initial version.